source/codegen/metadata/nirfmxspecan_restricted/CHANGES.md

```diff
@@ -0,0 +1,6 @@
+# Changes in metadata 
+
+## config.py
+
+The following metadata was added:
```

Comment:
Is this here because it is a change from the base configuration? In which case, should this also mention the new "restricted_driver_service" tag? Or is this just left over from the component the file was copied from?

Reply:
I think so. I copied the corresponding nirfmxspecan file. I added restricted_driver_service in commit 
c22188b.

Reply:
@madrasm : TL;DR it doesn't make since to have a "CHANGES.md" when there's no original source for it to have changed from. This file should be deleted.

@reckenro @DavidCurtiss

The original purpose of these files was to track any divergence from the hapigen metadata for MI drivers. I'm not sure that they were useful for that purpose (i.e., compared to looking at git history or writing a diff script) and I never used them when I worked on non-MI drivers. . We also have a different solution for tracking breaking_changes (pr tags).

The big test of the original intent of these drivers will be in David's feature to enable continuous updates. Have those notes been helpful? Maybe someone has discovered some other value for them?

If not, can we delete them? Or all of the non-hapigen ones?

Reply:
I didn't realize they were here. Either way, we can delete them. If we decide they'll be helpful, we can get them from the history.

Reply:
Deleted the changes.md I added in commit [a5c4b90](https://github.com/ni/grpc-device/pull/689/commits/a5c4b905d7062c775a6b492a7571544fab9620ba).

Reply:
Yeah, I think it was semi-useful when we were adding new MI drivers early on to kind of see what kind of changes the other driver made in one place. But as of now, I'd just look at the git history or diff against nimi-python versions of the files if I was curious. So from my perspective we should be fine deleting them all.

Reply:
I'll make a quick PR to clear out the rest.